optimum/intel/openvino/configuration.py

```diff
@@ -46,18 +46,48 @@
         "dataset": "wikitext2",
         "awq": True,
     },
-    "meta-llama/Llama-2-7b": {"bits": 4, "sym": True, "group_size": 128, "ratio": 0.7},
+    "meta-llama/Llama-2-7b": {"bits": 4, "sym": True, "group_size": 128, "ratio": 0.6},
     "meta-llama/Llama-2-7b-chat": {"bits": 4, "sym": True, "group_size": 128, "ratio": 0.8},
     "meta-llama/Llama-2-13b-chat": {"bits": 4, "sym": True, "group_size": 64, "ratio": 0.8},
-    "stabilityai/stablelm-3b-4e1t": {"bits": 4, "sym": True, "group_size": 64, "ratio": 0.8},
+    "stabilityai/stablelm-3b-4e1t": {
+        "bits": 4,
+        "sym": True,
+        "group_size": 64,
+        "ratio": 0.8,
+        "dataset": "wikitext2",
+        "awq": True,
+    },
     "stablelm-epoch-3b-preview": {"bits": 4, "sym": True, "group_size": 64, "ratio": 0.8},
```

Comment:
can this line be removed, it looks like it is accidentally copied with wrong naming format?

Reply:
Removed. Thanks!